src/mca/ptl/usock/ptl_usock.c

```diff
@@ -566,7 +566,7 @@ void pmix_usock_send_handler(int sd, short flags, void *cbdata)
     pmix_peer_t *peer = (pmix_peer_t*)cbdata;
     pmix_ptl_send_t *msg = peer->send_msg;
     pmix_status_t rc;
-    uint32_t nbytes;
+    size_t nbytes;
```

Comment:
@rhc54 note, that effectively this change is breaking backward compatibility with previous versions as we discussed in https://github.com/pmix/pmix/pull/1239.

Reply:
I mean not this one, but the one that caused it in #1239

Reply:
I'm looking at this PR now - I am not convinced we want this last commit as we cannot backport it to the release branches to make them work. Will have to ponder and look at alternatives.